Dockerfile

```diff
@@ -54,15 +54,15 @@ RUN apt-get update > /dev/null && \
 COPY install-mambaforge.bash /tmp/install-mambaforge.bash
 RUN /tmp/install-mambaforge.bash
 
+RUN chown -Rh rstudio:rstudio /home/rstudio
+RUN chown -Rh rstudio:rstudio /srv/conda
+
```

Comment:
If the /srv/conda chown is necessary, should the mambaforge install just be performed as NB_USER ? Then you could skip the recursive chown.

I'm surprised that it is necessary to chown /home/rstudio though.